service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/request/EhrExtractRequestHandler.java

```diff
@@ -103,4 +109,24 @@ private String getRequiredValue(Document xml, String xpath) {
         }
         return value;
     }
+
+    public EhrExtractStatus getEhrExtractStatus(GetGpcStructuredTaskDefinition structuredTaskDefinition) {
```

Comment:
These methods don't belong in EhrExtractRequestHandler. The purpose of this class is to handle the "EHR Extract Request" - a specific message we receive from the requesting practice via Spine. It is not related to to fetching the patient's data from GP connect